### PR TITLE
new aqt demo

### PR DESCRIPTION
**Pull request by singular-value**
From new notebook @richrines1 made

### CODE REVIEW

**Review by vtomole — DISMISSED** [review]
One minor nit then good to merge.

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-general-superstaq~=0.2.4
+general-superstaq~=0.2.5
 matplotlib~=3.0
-qiskit>=0.35.0, <0.37.0
+qiskit>=0.35.0, <0.37.2
```
Comment (richrines1) [inline requirements.txt:3]: why this change? we don't support qiskit>=0.37.0 atm

**Review by vtomole — APPROVED** [review]
LGTM
